docs/Signing.md

```diff
@@ -154,28 +157,33 @@ Root element. No options.
 
 A file to be signed.
 
-**Path** - file path, relative to the file path. If nested in a `<Container>`, is relative to the organization within the container
+**Path** - file path, relative to the file path. If nested in a parent element, is relative to the organization within the containing package
```

Comment:
`file path, relative to the file path`. Have I just not had enough coffee or is this wrong?